Kiosk/App/AppDelegate+GlobalActions.swift

```diff
@@ -207,18 +231,9 @@ private extension AppDelegate {
     func showWebController(address: String) -> Observable<Void> {
         return hideWebViewController().then (
             Observable.create { observer in
-                let webController = ModalWebViewController(url: NSURL(string: address)! as URL)
-
-                let nav = UINavigationController(rootViewController: webController)
-                nav.modalPresentationStyle = .formSheet
-
-                ARAnalytics.event("Show Web View", withProperties: ["url" : address])
-                self.window?.rootViewController?.present(nav, animated: true) {
+                self.showUserWebViewAtAddress(address) {
```

Comment:
While testing this, it appears that the normal flows for Privacy Policy and COS aren't working anymore, so I need to look into that before merging.